it/mail/chrome/messenger/messenger.properties

```diff
@@ -313,31 +313,31 @@ mailnews.search_date_format=5
 # separator for search date (e.g. "/", "-"), or empty when search_date_format is zero
 mailnews.search_date_separator=
 # leading zeros for day and month values, not used if mailnews.search_date_format is not zero
-mailnews.search_date_leading_zeros=vero
+mailnews.search_date_leading_zeros=true
 
-# offline msg 
+# offline msg
 nocachedbodybody=Il corpo di questo messaggio non è stato scaricato dal server per poter essere letto non in linea. Per leggerlo, è necessario riconnettersi alla rete, selezionare Non in linea dal menu File e quindi togliere il segno di spunta da "Lavora non in linea". In futuro è possibile scegliere quali messaggi o le cartelle che si desidera leggere non in linea, selezionando la voce Sincronizza dal menu File - Non in linea. È possibile impostare la preferenza Spazio su disco per prevenire la ricezione di messaggi troppo grandi.
 
 # accountCentral
 mailnews.account_central_page.url=chrome://messenger/content/msgAccountCentral.xul
 # LOCALIZATION NOTE(acctCentralTitleFormat): %1$S is brand, %2$S is account type, %3$S is account name
 acctCentralTitleFormat=%1$S %2$S - %3$S
-mailAcctType=Posta
-newsAcctType=Gruppi di discussione
+mailAcctType=E-mail
+newsAcctType=News
```

Comment:
I would translate "News", because in the source text it is specified "News" and not "Newsgroup"